tests/wpt/metadata/FileAPI/blob/Blob-constructor.html.ini

```diff
@@ -5,32 +5,20 @@
     bug: https://github.com/servo/rust-mozjs/issues/269
 
   [ArrayBuffer elements of the blobParts array should be supported.]
```

Comment:
The names of the subtests that now pass should be removed too.

Reply:
What happens if a test ends up without subtests in its .ini file? should I leave the file?

Reply:
Nope! The file can be removed.